pyp5js/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+pyp5js
+Copyright (C) 2019-2021 The pyp5js Contributors
```

Comment:
I'd change this to `Copyright (C) 2019-2021 Bernardo Fontes` because pyp5js still has no organization or something to group the contributors. This is something I'd like to do in the future, but I'm waiting for the project's community to become bigger. 

 Possibly this command line can help you to replace this:
 
 ```
 for x in `ack "The pyp5js Contributors"`; do sed -i "s/The pyp5js Contributors/Bernardo Fontes/g" $x; done;
 ```

Reply:
Done! Thanks for helping out with the command.